### DISCUSSION TITLE
sync vsplit views

### DISCUSSION THREAD

**Discussion by hernan3009** [helix-editor/helix#10937] [Ideas]
I would like to know if it is already possible or if there is a possibility of implementing the following functionality:

Split a text file (e.g., using vsplit) and keep the two views in sync. This means that the first line in the right view corresponds to the line immediately following the last line in the left view.

In many cases this would allow having a larger number of code lines visible at all times.

**Comment by KenCox94** [comment]
If its alright id like to look into this. 

Maybe like:
`:vsplit-extended` similar to `:vsplit-new`  ?

**Comment by Pronoiai** [comment]
I have the code in https://github.com/helix-editor/helix/pull/10934.
However preference appears to be make this a plugin later
